data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,265 @@ event "remnant: pantica void sprites"
 event "remnant: cognizance timer"
 	system "Nenia"
 		fleet "Ember Waste Node" 6000
+
+
+
+mission "Remnant: Expanded Horizons Astral 1"
+	name "Preparation for expedition"
+	description "To prepare for an expedition to Sagittarius A*, go to <destination> with 6 outfit space available."
+	passengers 1
+	minor
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Expanded Horizons Quarg 3: done"
+		# random < 25
+	on offer
+		conversation
+			`In the starport cafeteria you come across Dawn sitting at a table looking worried. She smiles when she sees you, and waves you to the seat opposite her. "Back from gallivanting around the galaxy with more tales to share?" she signs.`
+			choice
+				`	"Yes, as a matter of fact..."`
+				`	"No, just hunting Korath..."`
+				`	"No, just testing some stuff..."`
+				`	"You look worried. Is something happening?"`
+					goto worried
+			`	She listens for a minute or two as you tell her about what you have been up to, but it is obvious that she has something on her mind.`
+			choice
+				`	"You look worried. Is everything alright?"`
+				`	"You look distracted. Do you want to talk about it?"`
+			label worried
+			`	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She frowns, contemplating something unseen. "I should back up a step. While I do not have a permanent position yet, I am currently serving with an astrophysics team. We have been studying a variety of things, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`
```

Comment:
`	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She frowns, contemplating something unseen. "I should back up a step. Since we last talked, I was assigned to serve with an astrophysics team. My current work is part of the quantum condensate fabric density project, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`

OK, revised the paragraph to remove some of the ambiguity and clarify a few things. 

Amusingly, I've had very similar conversations with people, where I was in the same position as <first>. Is it frustrating? Yes. Is it confusing? Yes. Is it realistic and not conveniently parsed for optimum understanding? Yes. Is it the intended effect? Yes. At least to start with, anyway.  The intent was that the first bit of the paragraph is frustrating, like it usually is when one starts talking to someone who is several hours into a complex train of thought. The remainder is supposed to provide some background and clarify things a bit.

> 

> > I keep feeling like there's something I'm missing.
> 
> Missing from what? What are we talking about?

The astrophysical data, presumably. But she doesn't know if it is a problem with the data, or some sort of problem revealed by the data. She just knows she's picking up on something not jiving the way she thinks it should.

> > I do not have a permanent position yet...
> 
> A permanent position for what?

A career, a job, a work placement, somewhere to work proffessionally...
 
> > serving with an astrophysics teams
> 
> What are they doing?
> 
> > studying a variety of things
> 
> What things?

the quantum condensate fabric density project

> > prickling feeling that there's a problem that I am missing
> 
> Any ideas?

I'm sorry, I'm not an applied astrophysicist from the 31st century, I don't know exactly what sort of thing would be considered cutting edge research. I'm trying to avoid techno-babble where possible, but I guess this is one place where some is necessary. 

> > The senior researcher told me that most of the team feels it. There are some concrete theories, but nothing definitive
> 
> Feel what? Theories on what?
> 
> > Being able to take part in these expeditions is something I have been looking forward to for ages
> 
> What expeditions?
> 
> > I am currently putting together an expedition to Sagittarius A*
> 
> Why?

`to collect some fresh data I can use to check our records` (added text to explain this)

Reply:
If the intention is that it's frustrating then I'd expect a player choice to express that.

Reply:
Before I start doing that, do the changes help alleviate enough of the potential ambiguity? There's a fair bit more detail than there used to be.